src/shared/components/ChannelCover/ChannelCover.style.ts

```diff
@@ -0,0 +1,226 @@
+import { breakpoints, colors, sizes, transitions, typography, zIndex } from '@/shared/theme'
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+import { fluidRange } from 'polished'
+import Avatar from '../Avatar'
+import Button from '../Button'
+
+const SM_TITLE_HEIGHT = '44px'
+const TITLE_HEIGHT = '51px'
+const SM_SUBTITLE_HEIGHT = '24px'
+const SUBTITLE_HEIGHT = '27px'
+
+type CoverImageProps = {
+  src: string
+  editable?: boolean
+}
+
+type EditableOverlayProps = {
+  withImage?: boolean
+}
+
+export const Header = styled.section`
+  position: relative;
+`
+
+export const MediaWrapper = styled.div`
+  margin: 0 calc(-1 * var(--global-horizontal-padding));
+  height: 280px;
+  width: calc(100% + calc(2 * var(--global-horizontal-padding)));
+  position: relative;
+`
+
+export const Media = styled.div`
+  width: 100%;
+  height: 400px;
+  position: relative;
+  z-index: ${zIndex.background};
+`
+
+export const CoverImage = styled.div<CoverImageProps>`
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+
+  background-repeat: no-repeat;
+  background-position: center;
+  background-attachment: local;
+  background-size: cover;
+
+  background-image: linear-gradient(0deg, black 0%, rgba(0, 0, 0, 0.8) 30%, rgba(0, 0, 0, 0) 100%),
+    url(${({ src }) => src});
+  transition: opacity ${transitions.timings.loading} ${transitions.easing};
+  opacity: ${({ editable }) => (editable ? 0.6 : 1)};
+`
+
+const commonButtonStyles = css`
+  border: none;
+  background: none;
+  font-size: ${typography.sizes.subtitle2};
+  cursor: pointer;
+  position: absolute;
+`
+
+export const EditableOverlay = styled.div<EditableOverlayProps>`
+  z-index: 1;
+  width: 100%;
+  position: absolute;
+  display: flex;
+  height: 100%;
+  justify-content: center;
+  top: 0;
+  :hover button {
+    opacity: 1;
+  }
+`
+
+export const RemoveCoverButton = styled.button`
+  ${commonButtonStyles};
+  opacity: 1;
+  transition: opacity ${transitions.timings.loading} ${transitions.easing};
+  position: absolute;
+  display: flex;
+  align-items: center;
+  color: ${colors.white};
+  right: ${sizes(5)};
+  top: ${sizes(4)};
+  span {
+    margin-left: 10px;
+  }
+  svg {
+    width: 16px;
+    fill: ${colors.white};
+  }
+  @media screen and (min-width: ${breakpoints.small}) {
+    opacity: 0;
+  }
+  @media screen and (min-width: ${breakpoints.medium}) {
+    right: ${sizes(10)};
+    top: ${sizes(9)};
+  }
+`
+
+export const EditCoverButton = styled.button`
+  ${commonButtonStyles};
+  transition: opacity ${transitions.timings.loading} ${transitions.easing};
+  color: ${colors.white};
+  height: 100%;
+  width: 100%;
+  display: flex;
+  align-items: flex-end;
+  opacity: 1;
+  justify-content: center;
+  padding-bottom: 30px;
+
+  svg {
+    padding: 10px;
+    margin-right: 12px;
+    border-radius: 100%;
+    width: 40px;
+    border: 1px solid ${colors.white};
+    fill: ${colors.white};
+  }
+  span {
```

Comment:
I believe this needs some position tweaks in mobile view - text and svg are not centered

![deploy-preview-319--atlas-storybook netlify app__path=_story_shared-textarea--default](https://user-images.githubusercontent.com/59691518/108031582-667b5800-7031-11eb-9054-0ab415470a63.png)